.github/dependabot.yml

```diff
@@ -1,12 +1,14 @@
 version: 2
 updates:
 - package-ecosystem: "pip" # See documentation for possible values
-  directory: "pyproject.toml" # Location of package manifests
+  directory: "/"
```

Comment:
I noticed that there were no pip updates on this repository. I used this PR to fix it as well